fcrepo-kernel-modeshape/src/main/resources/fedora-node-types.cnd

```diff
@@ -73,7 +74,9 @@
   - premis:hasMessageDigest (URI) COPY
 
 [fedora:Skolem] > mix:referenceable mixin
+  - fedora:lastModified (DATE)
 
 [fedora:Pairtree] mixin
+  - fedora:lastModified (DATE)
```

Comment:
I am surprised that we want `fedora.lastModified` on Pairtrees. Is this an artifact of the previous logic of using `node.getParent()` instead of `resource.getContainer()`?